test/integration/speech-to-text.test.js

```diff
@@ -177,9 +177,9 @@ describe('speech to text integration', () => {
         customizationId,
         corpusName: 'test_corpus_3',
         corpusFile: fs
-          .readFileSync(path.join(__dirname, '../resources/speech_to_test/corpus-short-2.txt'))
+          .readFileSync(path.join(__dirname, '../resources/speech_to_text/corpus-short-2.txt'))
           .toString(),
-        allow_overwrite: true,
+        allowOverwrite: true,
```

Comment:
this feels like a breaking change

Reply:
This is fixing a test that was failing because the actual param is `allowOverwrite`. This was not picked up before because the node-sdk did not validate params in the past

Reply:
Ok sounds good